libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/ResourceMultiLanguageGenerator.cs

```diff
@@ -26,6 +29,12 @@ public ResourceMultiLanguageGenerator(string resourceId = null)
             this.ResourceId = resourceId;
         }
 
+        /// <summary>
+        /// Gets or sets the resource id.
+        /// </summary>
+        /// <value>
+        /// resource id.
```

Comment:
Could we change it to title case?
```suggestion
        /// Resource id.
```